Generator/Sources/NeedleFramework/Parsing/Pluginized/FileFilters/PluginizedContentFilter.swift

```diff
@@ -50,6 +50,14 @@ class PluginizedContentFilter: FileFilter {
         if containsNonCoreComponentInheritance {
             return true
         }
+        let containsDependencyInheritance = (Regex(": *Dependency").firstMatch(in: content) != nil)
+        if containsDependencyInheritance {
+            return true
+        }
+        let containsPluginExtensionInheritance = (Regex(": *PluginExtension").firstMatch(in: content) != nil)
+        if containsPluginExtensionInheritance {
+            return true
+        }
```

Comment:
Please see comment above this!

Reply:
Same here. I don't think we should change.